js/web/plunderer/js/plunderer.js

```diff
@@ -14,6 +14,10 @@
  */
 
 let plunderDB = new Dexie("PlayerDB");
+plunderDB.version(7).stores({
+	players: 'id,date',
```

Comment:
@dsiekiera  this is list of indexed for indexDB. If you want to run `.where()` than you need to declare index (You can filter without indexes but without speed boost).
Also when you change index you shoul add new version and keep previous to make Dexie happy.

Reply:
Yes, i found the solution. Its fixed in my branch already.